LICENSE

```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    <one line to give the program's name and a brief idea of what it does.>
-    Copyright (C) <year>  <name of author>
+    NewPipe a free lightweight YouTube frontend for Android
+    Copyright (C) 2020 TeamNewPipe
```

Comment:
We are `Team NewPipe`, not `TeamNewPipe`. The repo url is `TeamNewPipe` but if you go it, you'll see `Team NewPipe` as name.